charts/traction/templates/acapy/deployment.yaml

```diff
@@ -0,0 +1,178 @@
+{{- $globalName := include "global.name"  . -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "acapy.fullname" . }}
+  labels:
+    {{- include "acapy.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.acapy.autoscaling.enabled }}
+  replicas: {{ .Values.acapy.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "acapy.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.acapy.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "acapy.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.acapy.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "acapy.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.acapy.serviceAccount.automountServiceAccountToken }}
+      securityContext:
+        {{- toYaml .Values.acapy.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: register
+          image: curlimages/curl
+          imagePullPolicy: IfNotPresent
+          command:
+          - /bin/sh
+          args: [
+           "-c",
+           "{{ include "acapy.registerLedger" . }}"
+          ]
+          env:
+            - name: ACAPY_WALLET_SEED
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "acapy.fullname" . }}
+                  key: seed
+      containers:
```

Comment:
@usingtechnology is this container still needed? I assume it is what was being used to register DIDs on the ledger, but we switched to using the endorser request instead - correct?

Reply:
I wasn't aware of the switch. Was just moving it out of here https://github.com/bcgov/traction/blob/a3e68e952ad1baf2489377f06f4790054a9a7beb/charts/traction/templates/acapy_deployment.yaml#L39 into its own container.

Reply:
Np, just leave it there for now - wanted to confirm with @usingtechnology so we can pull this out in the nxt iteration

Reply:
i would assume it is still needed (this is a new layout for me)... unless you are starting on multiledger? in which case i guess you wouldn't want to register? that API that is called may not be valid for all ledgers anyway